### PR TITLE
feat: add pagination to call for more photos

### PR DESCRIPTION
**Pull request by haddadsim**
- Implementing button that renders more photos
- Implementing design
- Refactored api calls and put in a folder of its own

### CODE REVIEW

**Comment by haddadsim** [comment]
-Implemented show more button to render more photos

**Comment by alexiakl** [comment]
Nice work Simon! few comments:
- `.eslintrc` file is committed with an additional space character, make sure to remove it
- Load more button is above pictures, it should have its own space below pictures
- When clicking the close button X in Lightbox, the page is scrolling to the top
- On hover images, I recommend not resizing the size of the image div as visually this is looking bad
- Make sure the query to the API is correct, as the results do not always make sense.

**Comment by haddadsim** [comment]
> Nice work Simon! few comments:
> 
> * `.eslintrc` file is committed with an additional space character, make sure to remove it
> * Load more button is above pictures, it should have its own space below pictures
> * When clicking the close button X in Lightbox, the page is scrolling to the top
> * On hover images, I recommend not resizing the size of the image div as visually this is looking bad
> * Make sure the query to the API is correct, as the results do not always make sense.

- Removed extra space character from .eslintrc file
- Fixed location of show more button to not overlap with the images
- After clicking the close button in lightbox, the page stays where the photo was initially pressed on
- Removed the resizing of the images on hover.

File: src/components/SearchPage/SearchPage.jsx [hunk 1]
```diff
@@ -1,39 +1,53 @@
...
-const API_KEY = process.env.REACT_APP_ACCESS_KEY;
+import fetchImages from '../../utils/index';
 
 const SearchPage = () => {
   const [searchText, setSearchTerm] = useState('');
-  const [image, setImage] = useState(null);
+  const [image, setImage] = useState();
   const [isLoaded, setIsLoaded] = useState(false);
+  const [isNext, setIsNext] = useState(false);
+  const [nextPageIndex, setNextPageIndex] = useState(1);
```
Comment (george-haddad) [inline src/components/SearchPage/SearchPage.jsx:12]: Can you please explain why `useState()` is sent "false" in one statement and then "1" in the other?

Comment (haddadsim) [reply src/components/SearchPage/SearchPage.jsx:12]: They are 2 different states, IsNext is used to know when show more button is pressed.
NextPageIndex is used to know what page to query the Api, and the default value is 1.

Also in React-hooks, useState is used to initially set values for your states, and they can hold any kind of data (objects, arrays, integers etc)

Comment (george-haddad) [reply src/components/SearchPage/SearchPage.jsx:12]: Ah ok, it's being imported form `react` when I checked out the code it made more sense when looking at the whole file.

**Review by george-haddad — APPROVED** [review]
I will approve, but it's better you get another approval as well.

**Pull request merged**